frontend/src/app/trees/forests/tree-guidelines/tree-guidelines.component.html

```diff
@@ -1 +1,49 @@
-<app-sidebar-view *ngIf="template === 'sidebar'" [forest]="forest"></app-sidebar-view>
\ No newline at end of file
+<div class="usa-overlay" [ngClass]="{'is-visible' : showMobileNav}"></div>
+<button id="mobile-menu-btn" class="usa-menu-btn tree-guidelines-menu-btn" (click)='toggleMobileNav()' [style.position]="position" [style.top]="top">Menu</button>
+<nav role="navigation" [ngClass]="{'is-visible' : showMobileNav}" class="usa-nav sidenav-mobile">
```

Comment:
See the note above about refactoring this into a component

Reply:
@lauraGgit created story: https://github.com/nciinc/fs-intake-module/issues/124